src/fsharp/FSharp.Core/map.fsi

```diff
@@ -350,10 +352,12 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Count")>]
         val count: table:Map<'Key,'T> -> int
 
-        /// <summary>The keys in the map.</summary>
+        /// <summary>The keys in the map.
+        /// The sequence will be ordered by the keys of the map.</summary>
```

Comment:
If you'd like to add examples for each new function and method, that would be great.  We want to start adding them as routine.  See `option.fsi` for how to do this.

Reply:
yes, I'll do this, just need to find some time to do this